data/json/mapgen/cave.json

```diff
@@ -34,9 +34,9 @@
       ],
       "terrain": {
         ".": [ [ "t_region_groundcover_forest", 4 ], [ "t_region_tree", 1 ] ],
-        "|": "t_rock",
-        ",": "t_rock_roof",
-        "%": [ "t_rock", "t_region_groundcover_forest" ],
+        "|": "t_soil",
+        ",": "t_dirt_underground",
+        "%": [ "t_soil", "t_region_groundcover_forest" ],
```

Comment:
I think this is wrong. The "walls" of a cave shouldn't consist of soil. This OMT is an enter to rocky cave above ground, not its underground part.